R/tbl-format-footer.R

```diff
@@ -191,6 +182,50 @@ wrap_footer <- function(footer, setup, lines = setup$max_footer_lines, ellipsis
   map_chr(split, paste, collapse = " ")
 }
 
+wrap_footer_bullet <- function(footers, setup, lines = setup$max_footer_lines, ellipsis = TRUE) {
```

Comment:
Could we use cli for this?

Reply:
In principle, yes. Not sure how to keep the `#` comments.